content/microcontrollers/bbc-microbit.md

```diff
@@ -24,8 +24,12 @@ The micro:bit comes with the [DAPLink bootloader](https://tech.microbit.org/soft
 
 - Plug your micro:bit into your computer's USB port.
 - The micro:bit board will appear to your computer like a USB drive.
-- Determine the path to the board, on Linux it will be something like `/media/[USERNAME]/[NAME OF THE BOARD]`.
-- Build your TinyGo program to the board in .hex format using `tinygo build -o=/media/[USERNAME]/[NAME OF THE BOARD]/flash.hex -target=microbit [PATH TO YOUR PROGRAM]` command.
+- Determine the path to the board
+    - on Linux it will be something like `/media/[USERNAME]/[NAME OF THE BOARD]`.
+    - on Mac it is `/Volumes/[NAME OF THE BOARD]`.
+- Build your TinyGo program to the board in .hex format 
```

Comment:
"to the board" -> "for the board"

Reply:
In this case it is both building for the board (target) as well as TO the board (destination directory). So I let it go, as is.